### ISSUE TITLE
Automatically update locale for submission metadata when switching only locale

### ISSUE THREAD

**Issue by jnugent** [pkp/pkp-lib#5168]
We've recently had an installation with two journals with lots of published content switch from en_US to another single locale, and then disable the en_US locale afterwards.  The front end of the journal remains fine because even though the submission metadata is still en_US locale, the getLocalized versions of the various accessor methods default to whatever is available as a last resort.  

The back end, however, results in all of the submission metadata forms in the workflow showing with no content, since the new locale has no metadata and the old en_US locale is now disabled.  With the old locale disabled it isn't possible to copy/paste from the old locale either since it does not show in the multilingual fields.

I think there should at least be a warning if you disable your only enabled locale for a journal, and possibly provide a mechanism for switching localized metadata from the old locale to the new one, at least for things like title, abstract, and author settings as well.

Obviously the onus is probably on the journal to realize the problems with disabling their only previous locale.  We ended up writing a CLI tool to fix this, except for subjects and keywords because controlled vocabularies contained a mix of  languages across multiple journals.

**Comment by asmecher** [comment]
@jnugent, can you describe what they were trying to accomplish? Was the content in a non-English language, but entered into en_US fields? Or something else?

**Comment by jnugent** [comment]
@asmecher Correct.  They had non-English language content (articles, author details, etc) but were using the en_US locale for it, including the front end interface and the forms interfaces in the workflow.  Then they switched everything over to the non-English locale to bring everything into alignment with their content.

**Comment by asmecher** [comment]
That's going to necessitate essentially a database search-and-replace. I'm hesitant to do much coding for such a rare/wonky use case.

**Comment by jnugent** [comment]
That's essentially what my CLI tool did.  How about at least a warning message if they initially set up an install with only one locale, and then disable it in favour of another after creating content?   Even if it's just "This is a bad idea".

**Comment by asmecher** [comment]
Sure, a warning message wouldn't be a bad thing here.
